daemon/compartment.c

```diff
@@ -1340,6 +1365,13 @@ compartment_stop(compartment_t *compartment)
 
 	int ret = 0;
 
+	// check if container is stoppable
+	if (!compartment_is_stoppable(compartment)) {
+		ERROR("Compartment_stop not possible, as compartment in wrong state (%d)!.",
+		      compartment_get_state(compartment));
+		return -2;
+	}
+
```

Comment:
remove check completly. since container_is_stoppable() should be called from outside before calling the internal compartment_stop function.